src/textInput.test.jsx

```diff
@@ -18,7 +18,8 @@ describe('TextInput', function() {
 			id: NAME_ATTR,
 			maxLength: MAX_LEN,
 			error: ERROR_TEXT,
-			required: true
+			required: true,
+			onChange,
```

Comment:
one of the tricky aspects of relying on `beforeEach` is that it makes each unit test almost _too_ uniform - e.g. there's no test here that _doesn't_ set the optional `onChange` prop, so the tests missed out on the case there the component would fail if an `onChange` prop wasn't provided.

I think a good general rule might be to only provide _required_ props in `beforeEach` - every optional prop should be tested in isolation.